sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -35,14 +35,21 @@ const defaultConsumerClientOptions: Required<Pick<
 };
 
 /**
- * The `EventHubConsumerClient` is the main point of interaction for consuming events in Azure Event Hubs service.
+ * The `EventHubConsumerClient` is the main point of interaction for consuming events from an
+ * Event Hub instance.
  *
  * There are multiple ways to create an `EventHubConsumerClient`
  * - Use the connection string from the SAS policy created for your Event Hub instance.
  * - Use the connection string from the SAS policy created for your Event Hub namespace,
  * and the name of the Event Hub instance
- * - Use the fully qualified domain name of your Event Hub namespace like `<yournamespace>.servicebus.windows.net`,
+ * - Use the fully qualified version of the namespace like `<yournamespace>.servicebus.windows.net`,
  * and a credentials object.
+ * 
+ * Optionally, you can also pass the below:
+ * - An options bag to configure the retry policy or proxy settings.
+ * - A checkpoint store used to store checkpoints. These are the positions from where your application
```

Comment:
```suggestion
 * - A checkpoint store. These are the positions from where your application
```

Not sure what else you would do with a checkpoint store other than store checkpoints? 😂